core/services/chainlink/CONFIG.md

```diff
@@ -0,0 +1,114 @@
+# Configuration Transition
+
+- subgraph names are packages
+- thick lines indicate control flow
+- dotted lines indicate implicit interface implementation
+- regular w/ dot indicate implementation types
+
+```mermaid
```

Comment:
Nice! Was this generated or did you write it?

Reply:
:hand: :paintbrush: